dbServer/Dockerfile

```diff
@@ -7,6 +7,9 @@ RUN \
     # Check for mandatory build arguments
         : "${GRAPHDB_ZIP:?Provide the downloaded GraphDB Free ZIP file}"
 
+ENV GRAPHDB_WORKBENCH_EXTERNAL_URL=http://localhost/modelujeme/sluzby/dbServer
+ENV GRAPHDB_EXTERNAL_URL=http://localhost/modelujeme/sluzby/dbServer
```

Comment:
TBH I am not sure if this would work, as far as I know you cannot really override this ENV from the compose because it's part of the built image.